docs/source/guide/lre-4-low-level.md

```diff
@@ -0,0 +1,179 @@
+By altering the value for `noise_level`, ideal and noisy expectation values can be obtained.
+
+```{code-cell} ipython3
+from cirq import DensityMatrixSimulator, depolarize
+
+
+def execute(circuit, noise_level=0.025):
+    noisy_circuit = circuit.with_noise(depolarize(p=noise_level))
+    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
+    return rho[0, 0].real
+```
+
+Compare the noisy and ideal expectation values:
+
+```{code-cell} ipython3
+noisy = execute(circuit)
+ideal = execute(circuit, noise_level=0.0)
+print(f"Error without mitigation: {abs(ideal - noisy) :.5f}")
+```
+
+### Create noise-scaled circuits
+
+We start by creating a number of noise-scaled circuits which we will pass to the executor. {func}`.get_scale_factor_vectors` determines the scale factor vectors while {func}`.multivariate_layer_scaling` creates the noise scaled circuits. 
+
+Each noise-scaled circuit is scaled based on some pre-determined values of the scale factor vectors. These vectors depend
+on the degree of polynomial chosen for multivariate extrapolation and the fold multiplier for unitary folding.
+
+```{code-cell} ipython3
+import numpy as np
+from mitiq.lre.multivariate_scaling import get_scale_factor_vectors
+
+degree = 2
+fold_multiplier = 2
+
+scale_factors = get_scale_factor_vectors(circuit, degree, fold_multiplier)
+
+# print a randomly chosen scale factor vector
+
+print(f"Example scale factor vector: {scale_factors[2]}")
+
+```
+Each value in the scale factor vector corresponds to how unitary folding is applied to a layer in the circuit. If a scale factor value at some position in the scale factor vector is
+
+- greater than 1, then unitary folding is applied to the layer at that position in the circuit.
+- 1, then the layer at that position in the circuit is not scaled.
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import multivariate_layer_scaling
+
+noise_scaled_circuits = multivariate_layer_scaling(circuit, degree, fold_multiplier)
+num_scaled_circuits = len(noise_scaled_circuits)
+
+print(f"Total number of noise-scaled circuits for LRE = {num_scaled_circuits}")
+```
+
+An example noise-scaled circuit is shown below:
+
+```{code-cell} ipython3
+
+print("Example noise-scaled circuit:", noise_scaled_circuits[2], sep="\n")
+```
+
+### Evaluate noise-scaled expectation values
+
+With the many noise-scaled circuits in hand, we can run them through our executor to obtain the expectation values.
+
+```{code-cell} ipython3
+noise_scaled_exp_values = [
+    execute(circuit) for circuit in noise_scaled_circuits
+]
+```
+
+
+## Second step: Multivariate Extrapolation for the error-mitigated expectation value
+
+The penultimate step here is to fetch the coefficients we'll use to combine the noisy data we obtained above. Each noise scaled circuit has a coefficient of linear combination and a noisy expectation value associated with it. 
```

Comment:
This is probably a good place to put the equation for $O_\text{LRE}$ to help people understand what the coefficients are.